src/backend/updating-llvm.md

```diff
@@ -3,49 +3,90 @@
 <!-- toc -->
 
 <!-- date-check: Jul 2023 -->
-There is no formal policy about when to update LLVM or what it can be updated to,
-but a few guidelines are applied:
+Rust supports building against multiple LLVM versions:
 
-* We try to always support the latest released version
-* We try to support the last few versions
-  (and the number changes over time)
-* We allow moving to arbitrary commits during development
-* We strongly prefer to upstream all patches to LLVM before including them in rustc
+* Tip-of-tree for the current LLVM development branch is usually supported
+  within a few days. PRs for such fixes are tagged with `llvm-main`.
+* The latest released major version is always supported.
+* The one or two preceding major versions are usually supported.
 
-## Why update LLVM?
+By default, Rust uses its own fork in the [rust-lang/llvm-project repository].
+This fork is based on a `release/NN.x` branch of the upstream project, rather
```

Comment:
Could you change to `release/$N.x` and move "where `$N` is the version of LLVM that's being released" to here?